src/components/views/rooms/MessageComposer.tsx

```diff
@@ -602,7 +602,6 @@ export default class MessageComposer extends React.Component<IProps, IState> {
                     ) }
                     <div className="mx_MessageComposer_row">
                         { controls }
-                        { this.renderButtons(menuPosition) }
```

Comment:
not sure what this change is doing

Reply:
Fixed, ty for catching! Wrong merge probably.